src/components/ArticleTitles.jsx

```diff
@@ -0,0 +1,23 @@
+import { getArticles } from "../utils/api.js";
+import { useState, useEffect } from "react";
+
+const ArticleTitles = () => {
+
+    const [articles, setArticles] = useState ([])
+
+    useEffect(() => {
+        getArticles().then((response) => {
+            setArticles(response.data.articles)
+        })
+    }, [])
+
+    return (
+        <ul className="article-list"> 
+            {articles.map((article) => (
+                <li key={article.article_id}>{article.title}</li>
```

Comment:
From a UI perspective, it might be nice to return some more info, perhaps the date/time posted and/or author?